Fixed link to Twitter Application MAnagement

vstrien opened the pull request:
The link that should point to Twitter Application Management in reality pointed to the user @app. This correction fixes that and points to the "Applications"  page of Twitter where you can create Twitter apps.

azurecla commented:
Hi __@bhummerstone__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (benhu). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@bhummerstone : Thanks for your contribution to the Azure documentation! The author, @cherylmc, has been notified to review your proposed change.

cherylmc commented:
Looks good. Thanks, @bhummerstone .
#sign-off

vstrien committed:
Fixed link to Twitter Application MAnagement

The link that should point to Twitter Application Management in reality pointed to the user @app. This correction fixes that and points to the "Applications"  page of Twitter where you can create Twitter apps.
